content/requests/file_request_update_request.yml

```diff
@@ -0,0 +1,78 @@
+---
+title: File Request (Update)
+description: |-
+  The request body to update a file request.
+
+type: object
+
+properties:
+  title:
+    type: string
+    description: |-
+      An optional new title for the file request. This can be
+      used to change the title of the file request.
+
+      This will default to the value on the existing file request.
+    example: "Please upload required documents"
+
+  description:
+    type: string
+    description: |-
+      An optional new description for the file request. This can be
+      used to change the description of the file request.
+
+      This will default to the value on the existing file request.
+    example: "Please upload required documents"
+
+  status:
+    type: string
+    description: |-
+      An optional new status of the file request.
+
+      When the status is set to `INACTIVE`, the file request
```

Comment:
@virenderbox did we not agree on using lower case values for this in APIWG?

Reply:
We discussed in APIWG. However, decided to keep them this way as our V3 API uses upper case and is currently live. If this is a requirement, we can change it.

Reply:
The V2 styleguide uses lowercase for these kind of status strings and we should not deviate if we can fix this at the V2 level.

Reply:
Updated.